src/scikit_build_core/settings/skbuild_model.py

```diff
@@ -309,6 +309,18 @@ class WheelSettings:
     may not respect this exclusion.
     """
 
+    exclude_exts: List[str] = dataclasses.field(
+        default_factory=lambda: [".pyc", ".pyo"],
+        metadata=SettingsFieldMetadata(display_default="true"),
```

Comment:
```suggestion
        default=[".pyc", ".pyo"],
```
`display_default` will show that literal text in the documentation, I believe you don't want it to display `true` there right? Also, the lambda is unnecessary.